src/app/app.routing.module.ts

```diff
@@ -4,18 +4,18 @@ import { Routes, RouterModule } from '@angular/router';
 import { AuthGuard, UnauthGuard } from 'ngx-prx-styleguide';
 
 import { AuthorizationComponent } from './authorization/authorization.component';
-import { HomeComponent } from './home/home.component';
+import { DashboardComponent } from './dashboard/dashboard.component';
```

Comment:
Did some refactor/rename/move of some of these files from the original campaign list on the dashboard to fit the flight table within it.